assets/data/effects/wildermyth-drauven-pcs_outpostCelebration.json

```diff
@@ -0,0 +1,187 @@
+{
+"id": "wildermyth-drauven-pcs_outpostCelebration",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "wildermyth-drauven-pcs_outpostCelebration",
+	"modId": "wildermyth-drauven-pcs",
+	"author": "Morgan"
+},
+"type": "TIDING_MISC",
+"verb": "MANEUVER",
+"ability": {
+	"icon": "heroism",
+	"priority": "1",
+	"cooldown": "oncePerGame",
+	"encounterEnabled": true
+},
+"targets": [
+	{ "template": "EVENT" },
+	{ "template": "INJECTED_HERO" },
+	{ "template": "COMPANY" },
+	{
+		"role": "hero2",
+		"template": "HERO_BY_SCORE",
+		"scoreFunction": "drauven",
+		"scoreThreshold": "1",
+		"notAlreadyMatchedAs": [ "hero" ]
+	},
+	{
+		"role": "hero3",
+		"template": "HERO_BY_SCORE",
+		"scoreFunction": "SNARK",
+		"scoreThreshold": "60",
+		"notAlreadyMatchedAs": [ "hero2" ]
+	}
```

Comment:
🤔 Did you want this to only be able to fire if there is at least one Drauven Outpost, or just able to fire as long as there's at least one Drauven in the party?  If you like, we could add an additional required role using the `SITE` template to find one with the `drauvenOutpost` aspect.  Then you could also reference the location in the narration of the comic if you want.